vtex/faststore
# feat: add support for Section Override API V2 to ProductDetails

## What's the purpose of this pull request?

Adds support for Section Override API V2 to the ProductDetails section.

## How it works?

It now supports the SectionOverride context on all its components and remains compatible with Section Override API V1 as well.

## How to test it?

Navigate to the PLP in the deploy preview below, make sure the product details is functional and that the price is formatted in BRL. That's the change introduced with the override.

### Starters Deploy Preview
https://github.com/vtex-sites/starter.store/pull/323

## References

#2091
